src/main/java9/module-info.java

```diff
@@ -0,0 +1,19 @@
+module org.xerial.sqlitejdbc {
+
+    requires transitive java.logging;
+    requires transitive java.sql;
+    requires transitive java.sql.rowset;
```

Comment:
same here, there's no mention of this anywhere in the code, why do we need that?